app/components/common/drawer/index.tsx

```diff
@@ -34,7 +34,7 @@ const Drawer: React.FC<DrawerProps> = ({ event, isDrawerVisible, toggleDrawer })
           <h1 className=" font-semibold text-3xl leading-10 capitalize">{event.title}</h1>
           <EventVisibility visibility={event.visibility ?? ''} />
           <p className="py-2 text-base font-normal leading-5 text-left">
-            {moment(event.start).format('MMMM DD, YYYY h A')} - {moment(event.end).format('h A')}
```

Comment:
can we remove moment and add dayjs instead

Reply:
updated